File: projet/source/f1levier/app/src/main/java/fr/utbm/lo52/f1levier/db/entity/Participant.java
package fr.utbm.lo52.f1levier.db.entity;


import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity(tableName = "participant")
public class Participant {

    @PrimaryKey(autoGenerate = true)
    public int id;

    public String name;

    public int echelon;

    public Participant(@NonNull String name, int echelon) {
        this.name = name;
        this.echelon = echelon;
    }
}

File: TP4/CourseCODEP25/app/src/main/java/com/utbm/nouassi/manou/coursecodep25/database/entity/Participant.kt
package com.utbm.nouassi.manou.coursecodep25.database.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Participant (
    @PrimaryKey(autoGenerate = true)
    val id: Int,

    @ColumnInfo(name = "nom")
    val nom: String,

    @ColumnInfo(name = "niveau")
    val niveau: Int

 )